### DISCUSSION TITLE
react  antd为啥更新这么频繁

### DISCUSSION THREAD

**Discussion by animalgithub** [ant-design/ant-design#45264] [General]
为啥频繁的更新 不能好好打磨下

**Comment by RaoHai** [comment]
好好打磨不就是更新么。

**Comment by zombieJ** [comment]
[轮值规则和版本发布流程](https://github.com/ant-design/ant-design/wiki/%E8%BD%AE%E5%80%BC%E8%A7%84%E5%88%99%E5%92%8C%E7%89%88%E6%9C%AC%E5%8F%91%E5%B8%83%E6%B5%81%E7%A8%8B)

antd 每周发布 patch 更新，每月发布 minor 更新。如果不需要保持更新，可以考虑锁版本哈~

**Comment by yoyo837** [comment]
他可能想问的是为什么不每个版本间隔长一点，很简单，你一看就是没有被停止维护留下一堆大bug的开源项目教育过，已知问题对于用户来说恨不得希望马上获得修复才对。所以patch版本越快越好，minor，major适当间隔，毕竟主要是新增和变更内容。
